change/@fluentui-react-native-tokens-68266dc0-0012-4f00-85ba-03e4eed0710b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Kind of surprised you need all these change files... You might want to fetch origin to make the comparison more recent.

The only change files I would expect to see would be for the button package and the fluent-tester package